packages/cli/src/util/createServer.ts

```diff
@@ -50,18 +44,19 @@ export function createSingleProcessPrism(options: CreatePrismOptions) {
 }
 
 async function createPrismServerWithLogger(options: CreatePrismOptions, logInstance: Logger) {
-  const server = createServer(
-    options.spec,
-    { mock: { dynamic: options.dynamic } },
-    logInstance.child({ name: 'HTTP SERVER' }),
-  );
+  if (options.operations.length === 0) {
```

Comment:
This will be refactored in the future; I can force TypeScript to make sure that the operations is *at least* an array with an element. Not now though.